### PR TITLE
Enzyme/unit tests

### PR DESCRIPTION
**Pull request by hyukkim0304**
@allisonc07 ready for reviews!

### CODE REVIEW

File: client/src/components/total_number_reviews/total_number_reviews.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const React = require('react');
+
```
Comment (allisonc07) [inline client/src/components/total_number_reviews/total_number_reviews.js:2]: in your original file, you imported a css stylesheet! don't forget it if you need it :D

**Review by allisonc07 — COMMENTED** [review]
looks good! just a quick question, why did you change all your .jsx to .js?

File: client/src/components/total_number_reviews/total_number_reviews.js (hunk 1 above)
Comment (hyukkim0304) [reply client/src/components/total_number_reviews/total_number_reviews.js:2]: i removed it cuz it's throwing an error on tests! will add it later!

Comment (allisonc07) [reply client/src/components/total_number_reviews/total_number_reviews.js:2]: 👍

**Comment by hyukkim0304** [comment]
I don't know why but EsLint is angry about it..

**Comment by allisonc07** [comment]
Is it getting angry at the import statement? like 
`import Header from './Header.jsx';` is where it objects?

If it is, you can add 
`resolve: {
    extensions: ['.js', '.jsx'],
  },`

to your webpack.config.js in the module.exports! and then you can just do 
`import Header from './Header';` and it will resolve it for you :D

If it's getting angry elsewhere, then I'm not sure haha

**Review by allisonc07 — COMMENTED** [review]
looks good!

**Pull request merged**